Extending/Packages/Good-practice/index.md

```diff
@@ -0,0 +1,161 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Package development good practice"
+meta.Description: "Information on common defaults and good practice for umbraco package development"
+---
+
+# Good practice and defaults
+_This document provides some guides and notes on package development, and good practice that will help you maintain and support your package through multiple releases and versions of Umbraco. These good practices are not prescriptive, but mearly offer a guide as to what we have found works well, and not-so-well, when developing packages for Umbraco_
+
+## App_Plugins
```

Comment:
Should this also explain what does go into the App_Plugins folder? Something like: 'Each package should create a subfolder with the package name/id (e.g. `App_Plugins\MyPackage`) to store their backoffice related code, like a `package.manifest`, the AngularJS views/controllers, CSS and language files.'

And maybe also note that these files will be served publicly (even though this folder isn't located in the `wwwroot`)?